more details for examples script container (#20)
deephaven/examples: README.md

   1. `docker build -t deephaven/examples samples`
   2. `docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples download`

> The `docker build` command builds a Docker container which hosts a script that helps manage the examples. That script will automate the management of the example files, allowing you to download and update the example files without directly working with the git project hosting the examples. This command is only needed once; after it runs, the container is available to use on your host.

Reviewer: ```suggestion
The `docker build` command builds a Docker container containing a script that helps manage the examples. 
 That script will automate the management of the example files.  It allows you to download and update the example files without directly working with the git project hosting the examples.  This `docker build` command is only needed once; after it runs, the container is available to use on your host.
```

> The `docker run` command runs that new container, letting it know that the examples will end up in your `deephaven-core/docker/core/data` path. When Deephaven runs in its containers, this directory from the host is mounted inside the container as `/data/examples`.

Reviewer: ```suggestion
The `docker run` command downloads the example data by running the new examples management container.  The `-v "$(pwd)/docker/core/data:/data"` argument mounts your local `deephaven-core/docker/core/data` path as `/data` in the container.  The example data is stored to `/data/examples` inside the container, which is `deephaven-core/docker/core/data/examples` on the local filesystem.
```

> You can run `docker run` again to manage the example data -- for example, to download a new version, or remove the sample data. This command:
> 
> 	`docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples`
> 
> will run the contained script and show usage for the different commands it offers.

Reviewer: ```suggestion
You can run `docker run` again to manage the example data -- for example, to download a new version. To see what options are available, run:

	`docker run --rm -v "$(pwd)/docker/core/data:/data" deephaven/examples`
```
